src/services/ai-chat-service.ts

```
@@ -346,12 +359,13 @@ export class AIChatService {
     if (!this.provider) return null;
     return {
       provider: this.provider.name,
-      model: this.currentModel || this.provider.getDefaultModel()
+      model: this.currentModel || this.provider.getDefaultModel(),
     };
   }
 
-  // @ts-ignore - Used in future features  
-  private _formatSOWResponse(sow: any): string { // Legacy method - used in future features
+  // @ts-ignore - Used in future features
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)